tools/distpartitioning/utils.py

```diff
@@ -64,37 +95,26 @@ def get_node_types(schema):
         dictionary with ntype <-> type_nid mappings
         list of ntype strings
     """
-    #Get the node_id ranges from the schema
-    global_nid_ranges = schema['nid']
-    global_nid_ranges = {key: np.array(global_nid_ranges[key]).reshape(1,2)
-                    for key in global_nid_ranges}
-
-    #Create an array with the starting id for each node_type and sort
-    ntypes = [(key, global_nid_ranges[key][0,0]) for key in global_nid_ranges]
-    ntypes.sort(key=lambda e: e[1])
-
-    #Create node_typename -> node_type dictionary
-    ntypes = [e[0] for e in ntypes]
-    ntypes_map = {e: i for i, e in enumerate(ntypes)}
-
-    return ntypes_map, ntypes
+    ntype_info = schema["nid"]
+    ntypes = []
+    for k in ntype_info.keys(): 
+        ntypes.append(k)
+    ntype_ntypeid_map = {e: i for i, e in enumerate(ntypes)}
+    ntypeid_ntype_map = {str(i): e for i, e in enumerate(ntypes)}
+    return ntype_ntypeid_map, ntypes, ntypeid_ntype_map
```

Comment:
why is the return values of `get_node_types` different from `get_edge_types`?

Reply:
Removed get_edge_types. Basically this is an unused function. and in the current iteration we do not support edge features.